website/pages/en/index.js

```diff
@@ -59,62 +49,222 @@ const PromoSection = props => (
+      <PromoSection>
+        <Button href={docUrl("index.html", language)}>Get Started</Button>
+        <Button href={pageUrl("videos.html", language)}>Videos</Button>
+      </PromoSection>
+    </div>
+  );
+};
+
+const WorkSponsors = () => {
   return (
-    <div className="productShowcaseSection paddingBottom">
-      <h2>{"Who's Using Babel?"}</h2>
-      <div className="logos">{showcase}</div>
-      <div className="more-users">
-        <a className="button" href={pageUrl("users.html", props.language)}>
-          More {siteConfig.title} Users
-        </a>
+    <div
+      className="blockElement alignCenter twoByGridBlock sponsors-work"
+      style={{ flexBasis: "40%" }}
+    >
+      <h2>Friends of Open Source</h2>
+      <p style={{ fontSize: 16 }}>
+        These companies are being awesome and paying their engineers to work on
+        Babel
+      </p>
+      <div className="productShowcaseSection">
+        <div className="cards">
+          {siteConfig.sponsors
+            .filter(sponsor => {
+              return sponsor.type == "work";
+            })
+            .map((sponsor, i) => {
+              return (
+                <a className="card" href={sponsor.url} target="_blank" key={i}>
+                  <div className="card-image">
+                    <img src={sponsor.image} title={sponsor.name} />
+                  </div>
+                  <div className="card-text">
+                    <p>{sponsor.description}</p>
+                  </div>
+                </a>
+              );
+            })}
+        </div>
       </div>
     </div>
   );
 };
 
+const SponsorTier = props => {
+  const tierSponsors = siteConfig.sponsors.filter(
+    sponsor => sponsor.type == "opencollective" && sponsor.tier === props.tier
+  );
+  return (
+    <div>
+      <h3>{props.title}</h3>
+      <ul className={`sponsors-opencollective-tier tier-${props.tier}`}>
+        {tierSponsors.map((sponsor, i) => (
+          <li key={i}>
+            <a href={sponsor.url} title={sponsor.name}>
+              <img src={sponsor.image} />
+            </a>
+          </li>
+        ))}
+      </ul>
+      {props.button ? (
+        <PromoSection>
+          <Button href="https://opencollective.com/babel" target="_blank">
+            Become a sponsor
+          </Button>
+        </PromoSection>
+      ) : null}
+    </div>
+  );
+};
+
+const OpenCollectiveSponsors = props => {
+  const language = props.language || "en";
+  return (
+    <div className="container paddingTop paddingBottom">
+      <div className="wrapper productShowcaseSection">
+        <div className="support-the-team">
+          <h2>Support the team</h2>
+          <p>
+            Babel is helping shape the future of the JavaScript language itself,
+            being used at companies like Facebook, Google, Netflix, and{" "}
+            <a href={pageUrl("users.html", language)}>hundreds more</a>. Your
+            donation will help cover expenses like attending TC39 ( the
```

Comment:
remove the `( the` space here, and `meettings` is `meetings`.

Yeah don't know how many things we should put here since I can think of more like attending/speaking at events/meetups, doing contributor workshops. Or we should probably make a new page that explains more of this actually..

Reply:
For now I think a separate page would just become a duplicate of the opencollective page the donate button links to. Unless the new page will also have paypal or other ways to donate?

On the current homepage section I could use some help in writing a better summary to highlight the top expenses.